mappings/net/minecraft/client/render/entity/model/LargePufferfishEntityModel.mapping

```diff
@@ -1 +1,15 @@
 CLASS net/minecraft/class_592 net/minecraft/client/render/entity/model/LargePufferfishEntityModel
+	FIELD field_3487 bottomBackSpikes Lnet/minecraft/class_630;
+	FIELD field_3488 rearLeftSpikes Lnet/minecraft/class_630;
+	FIELD field_3489 rightFontSpikes Lnet/minecraft/class_630;
+	FIELD field_3490 topFrontSpikes Lnet/minecraft/class_630;
+	FIELD field_3491 bottomFontSpikes Lnet/minecraft/class_630;
+	FIELD field_3492 bottomSpikes Lnet/minecraft/class_630;
+	FIELD field_3493 body Lnet/minecraft/class_630;
+	FIELD field_3494 leftFin Lnet/minecraft/class_630;
+	FIELD field_3495 topBackSpikes Lnet/minecraft/class_630;
+	FIELD field_3496 topSpikes Lnet/minecraft/class_630;
+	FIELD field_3497 leftFrontSpikes Lnet/minecraft/class_630;
+		COMMENT
+	FIELD field_3498 rearRightSpikes Lnet/minecraft/class_630;
```

Comment:
i'd prefer order like left/right front/rear like you did in horse/hoglin model fields